Library/Homebrew/formulary.rb

```diff
@@ -251,4 +253,19 @@ def self.tap_paths(name)
       ]).detect(&:file?)
     end.compact
   end
+
+  def self.find_with_priority(ref, spec=:stable)
+    possible_pinned_tap_formulae = tap_paths(ref, Dir["#{HOMEBREW_LIBRARY}/PinnedTaps/*/*/"]).map(&:realpath)
```

Comment:
OK. Let's keep this as it is. For some reason, we prefer `resolved_path` but I have forgotten why.